lld/MachO/SyntheticSections.h

```diff
@@ -315,6 +315,25 @@ class StubHelperSection final : public SyntheticSection {
   Defined *dyldPrivate = nullptr;
 };
 
+class ObjCSelRefsSection final : public SyntheticSection {
+public:
+  ObjCSelRefsSection();
+  void initialize();
+
+  // This SyntheticSection does not do its own writing, but instead uses the
+  // creates SyntheticInputSection's and inserts them into inputSections
+  uint64_t getSize() const override { return 0; }
```

Comment:
It's odd to inherit `ObjCSelRefsSection` from `SyntheticSection` without using any features.
Can we just use a standalone class like `ObjCSelRefsHelper` to assist this selector reference?
Or, do you actually plan to write something in the future?

Reply:
Agree, I don't think there is any good current pattern for this. I followed closest available pattern `PageZeroSection` which also doen't use the features.
We can make it a global helper, but then I think it has to be static - also not a good pattern. Any suggestions where to put the helper ?